internal/controller/serving/inferenceservice_controller_test.go

```diff
@@ -1289,6 +1293,343 @@ var _ = Describe("InferenceService Controller", func() {
 			})
 		})
 	})
+
+	Describe("KServe KEDA Reconciler", func() {
+		var (
+			testNs         string
+			kedaReconciler *reconcilers.KserveKEDAReconciler
+		)
+
+		BeforeEach(func() {
+			ctx = context.Background()
```

Comment:
```suggestion
```

No need to overwrite that global variables here :)

Reply:
Done